lib/Extractor/ExprBuilder.cpp

```diff
@@ -376,6 +376,23 @@ Inst *ExprBuilder::getDemandedBitsCondition(Inst *I) {
                                    LIC->getInst(Inst::Eq, 1, {Res, Zero})});
     Result = LIC->getInst(Inst::And, 1, {Result, SignBits});
   }
+  // TODO: We might want to look into handling empty set later
+  if (!I->Range.isEmptySet() && !I->Range.isFullSet()) {
+    Inst *Lower = LIC->getConst(I->Range.getLower());
+    Inst *Upper = LIC->getConst(I->Range.getUpper());
+ 
+    // not wrapped set: x >= Lower && x <= Upper
```

Comment:
this comment isn't correct, it's x < Upper, right?
anyway, since the code below is clear, you can delete these lines